code/ImageGalleryPage.php

```diff
@@ -157,6 +157,8 @@ public function getCMSFields() {
 				)
 			);
 		}
+
+		$this->extend('updateCMSFields', $fields);
```

Comment:
You're right, called twice. But how to overcome the issue e.g. with fluent that relies that all fields are in place? Is it a huge problem (beside performance) that it's called twice?